cmd/kusk/manifests/api_server.yaml

```diff
@@ -113,7 +122,24 @@ spec:
         - image: kubeshop/kusk-gateway-api:v1.1.10
           imagePullPolicy: Always
           name: kusk-gateway-api
-      serviceAccountName: kusk-gateway-manager
+        - name: kusk-gateway-api-websocket
+          image: kubeshop/kusk-gateway-api-websocket:v1.1.10
```

Comment:
According to <https://hub.docker.com/r/kubeshop/kusk-gateway-api-websocket/tags>, there's no such image tagged `kubeshop/kusk-gateway-api-websocket:v1.1.10`.

Reply:
This needs updating to v1.2.1 as per my release today of the api and websocket